xbmc/guilib/GUIMessage.h

```diff
@@ -365,9 +365,15 @@ class CFileItemList;
 class CGUIMessage final
 {
 public:
-  CGUIMessage(int dwMsg, int senderID, int controlID, int param1 = 0, int param2 = 0);
-  CGUIMessage(int msg, int senderID, int controlID, int param1, int param2, CFileItemList* item);
-  CGUIMessage(int msg, int senderID, int controlID, int param1, int param2, const CGUIListItemPtr &item);
+  CGUIMessage(int dwMsg, int senderID, int controlID, int64_t param1 = 0, int64_t param2 = 0);
+  CGUIMessage(
+      int msg, int senderID, int controlID, int64_t param1, int64_t param2, CFileItemList* item);
+  CGUIMessage(int msg,
+              int senderID,
+              int controlID,
+              int64_t param1,
+              int64_t param2,
```

Comment:
Please add stdint.h inclusion to the header

Reply:
use c++ header <cstdint>